android/cli/commands/_buildModule.js

```diff
@@ -1474,6 +1474,11 @@ AndroidModuleBuilder.prototype.packageZip = function (next) {
 					}.bind(this));
 				}
 
+				// respackageinfo file
+				if (this.manifest.respackage) {
```

Comment:
Why add a `respackage` key to the manifest instead of just checking if the `respackageinfo` file exists?